Kernel/Term.hpp

```diff
@@ -92,7 +92,8 @@ bool operator<(const TermList& lhs, const TermList& rhs);
 class TermList {
 public:
   CLASS_NAME(TermList)
-  static const unsigned SPEC_UPPER_BOUND = 10000000;
+  // divide by 4 because of the tag, by 2 to split the space evenly
+  static const unsigned SPEC_UPPER_BOUND = (UINT_MAX / 4) / 2;
```

Comment:
It's fixed in master as well. The diff shows what the edit was back then not how it compares to master now (I was also confused). 

In any case, Github boldly claims it can do the merge. And it's hard to imagine that it could mess up given that both master and this branch insist on the new version...